gemrb/GUIScripts/GUICommonWindows.py

```diff
@@ -1721,6 +1721,8 @@ def UpdatePortraitWindow ():
 		if (pcID <= GemRB.GetPartySize()):
 			Button.SetAction(lambda btn, val, pc=pcID: GemRB.GameControlLocateActor(pc), IE_ACT_MOUSE_ENTER);
 			Button.SetAction(lambda: GemRB.GameControlLocateActor(-1), IE_ACT_MOUSE_LEAVE);
+		        if (i < 6):
+				Button.SetHotKey(chr(ord('1') + i), 0, True)
```

Comment:
I'm surprised python didn't exit complaining here, since the indentation is all wrong.

Reply:
I'm not the greatest Python expert - how is it wrong?

Reply:
You used spaces on the first line, while everything uses tabs. I guess you have a tab width of 2 set, so it's easier to make the mistake.

Reply:
Oh wow. That's emacs autoindenting it, and using spaces despite the fact that I thought I'd set it up not to (and only in place of one tab apparently).